lib/ReactTags.js

```diff
@@ -29,12 +29,16 @@ const CLASS_NAMES = {
   suggestionDisabled: 'is-disabled'
 }
 
+function findMatchIndex(options, query) {
```

Comment:
If you run the command `npx standard lib/*.js --fix` then code style issues will be automatically fixed ✨